pyccel/stdlib/stdlib.py

```diff
@@ -1,5 +1,11 @@
 # coding: utf-8
 
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

@saidctb should this file still exist?

Reply:
probably not